### PR TITLE
wip(email): feat #148 phpmailer

### PR DESCRIPTION
**Pull request by derchrisuk**
Implemented PHPMailer for email delivery using SMTP/Pop before SMTP/phpmail()

Settings available in admincenter, including debug.
Tested most of the modules, still a few tests left.
Only module which needs work is forum_topic, as not sure how to implement a good error/debug output.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL/pull/205%23issuecomment-73751681%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23issuecomment-73751886%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23issuecomment-73752025%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23issuecomment-73759007%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23issuecomment-85625606%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23issuecomment-88180465%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23issuecomment-90994177%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23issuecomment-90994774%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23discussion_r25248170%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23discussion_r27997756%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532199%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532201%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532205%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532208%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532209%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532212%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532771%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9533083%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-10630207%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-10630208%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-10630210%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-10630242%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/81b23018af8ed887de6b9dc84dafe36c035bcf3e%23commitcomment-9568700%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/commit/81b23018af8ed887de6b9dc84dafe36c035bcf3e%23commitcomment-10630211%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23issuecomment-73751681%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20still%20think%20that%20php%20mailer%20shoul%20be%20inside%20the%20components%20folder%20to%20make%20clear%20that%20it%20is%20not%20our%20part%20of%20code%22%2C%20%22created_at%22%3A%20%222015-02-10T18%3A13%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20agree%2C%20but%20I%20got%20some%20include%20errors%20when%20the%20module%20was%20not%20in%20the%20same%20folder%20as%20the%20class%22%2C%20%22created_at%22%3A%20%222015-02-10T18%3A14%3A51Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22But%20that%20should%20be%20resolveable%20%3B%29%20maybe%20a%20chdir%20into%20the%20components%20folder%20and%20than%20back%20after%20including%22%2C%20%22created_at%22%3A%20%222015-02-10T18%3A15%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-02-10T18%3A53%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-03-24T18%3A06%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20think%20I%20saw%20a%20problem%20with%20mails%20not%20being%20delivered%20as%20HTML.%5Cr%5CnNeed%20to%20check%20each%20email%20send%2C%20and%20what%20the%20actual%20email%20looks%20like.%5Cr%5CnOnce%20done%20can%20be%20merged.%20%22%2C%20%22created_at%22%3A%20%222015-03-31T17%3A33%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-04-08T18%3A20%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-04-08T18%3A22%3A08Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Commit%205e52444eb7f67dc243247b5ad71161572ff015ef%20src/func/email.php%2026%2062%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532199%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20do%20you%20put%20quotes%20here%3F%22%2C%20%22created_at%22%3A%20%222015-02-01T07%3A35%3A35Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22Should%20be%20single%20quotes%22%2C%20%22created_at%22%3A%20%222015-02-01T07%3A37%3A22Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20think%20there%20should%20be%20no%20quootes%20because%20it%20is%20already%20a%20string%22%2C%20%22created_at%22%3A%20%222015-02-01T07%3A38%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20was%20going%20by%20the%20official%20documentation%20and%20examples.%5Cr%5CnSome%20had%20double%2C%20some%20had%20single%20quotes.%22%2C%20%22created_at%22%3A%20%222015-02-01T07%3A39%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-04-08T18%3A20%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/func/email.php%3AL62%20%285e52444%29%22%7D%2C%20%22Commit%205e52444eb7f67dc243247b5ad71161572ff015ef%20src/func/email.php%2015%2052%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9533083%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20think%20that%20the%20namespace%20needs%20to%20be%20fixed%20there%20too%22%2C%20%22created_at%22%3A%20%222015-02-01T13%3A23%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-04-08T18%3A20%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/func/email.php%3AL52%20%285e52444%29%22%7D%2C%20%22Pull%2024c20346947334ca2e354ab3fd071b48de6d6b12%20composer.json%2021%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/205%23discussion_r25248170%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20should%20be%20the%20only%20%60require%60%20element%2C%20all%20codesniffer%2C%20test-reporter%20and%20phpunit%20are%20%5B%60require-dev%60%5D%28https%3A//getcomposer.org/doc/04-schema.md%23require-dev%29.%22%2C%20%22created_at%22%3A%20%222015-02-24T12%3A26%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-04-08T18%3A20%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20composer.json%3AL1-14%22%7D%2C%20%22Commit%205e52444eb7f67dc243247b5ad71161572ff015ef%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532201%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20do%20not%20think%20that%20is%20is%20good%20to%20move%20phpmailer%20there.all%20the%20external%20code%20should%20be%20in%20components%22%2C%20%22created_at%22%3A%20%222015-02-01T07%3A36%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20tried%20it%20within%20the%20components%20folder%2C%20but%20got%20error%20messages%20about%20file%20not%20found.%22%2C%20%22created_at%22%3A%20%222015-02-01T07%3A38%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-04-08T18%3A20%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22Commit%205e52444%20comments%22%7D%2C%20%22Commit%205e52444eb7f67dc243247b5ad71161572ff015ef%20admin/languages/uk/admincenter.php%204%2042%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef%23commitcomment-9532771%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Shoud%20be%20%60E-Mail%60%20http%3A//www.duden.de/rechtschreibung/E_Mail%22%2C%20%22created_at%22%3A%20%222015-02-01T11%3A36%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-04-08T18%3A21%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20admin/languages/uk/admincenter.php%3AL42%20%285e52444%29%22%7D%2C%20%22Commit%2081b23018af8ed887de6b9dc84dafe36c035bcf3e%20myprofile.php%2012%20417%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/commit/81b23018af8ed887de6b9dc84dafe36c035bcf3e%23commitcomment-9568700%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22echo%20is%20missing%22%2C%20%22created_at%22%3A%20%222015-02-03T19%3A03%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-04-08T18%3A20%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20myprofile.php%3AL417%20%2881b2301%29%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/derchrisuk%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%7D%2C%20%22https%3A//github.com/Phhere%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/derchrisuk'></a><a href='https://github.com/Phhere'></a>

- [x] <a href='#crh-comment-Commit 5e52444eb7f67dc243247b5ad71161572ff015ef src/func/email.php 26 62'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef#commitcomment-9532199'>File: src/func/email.php:L62 (5e52444)</a></b>
- <a href='https://github.com/Phhere'></a> Why do you put quotes here?
- <a href='https://github.com/derchrisuk'></a> Should be single quotes
- <a href='https://github.com/Phhere'></a> I think there should be no quootes because it is already a string
- <a href='https://github.com/derchrisuk'></a> I was going by the official documentation and examples.
Some had double, some had single quotes.
- [x] <a href='#crh-comment-Commit 5e52444eb7f67dc243247b5ad71161572ff015ef'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef#commitcomment-9532201'>Commit 5e52444 comments</a></b>
- <a href='https://github.com/Phhere'></a> I do not think that is is good to move phpmailer there.all the external code should be in components
- <a href='https://github.com/derchrisuk'></a> I tried it within the components folder, but got error messages about file not found.
- [x] <a href='#crh-comment-Commit 5e52444eb7f67dc243247b5ad71161572ff015ef admin/languages/uk/admincenter.php 4 42'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef#commitcomment-9532771'>File: admin/languages/uk/admincenter.php:L42 (5e52444)</a></b>
- <a href='https://github.com/Pascalmh'></a> Shoud be `E-Mail` http://www.duden.de/rechtschreibung/E_Mail
- [x] <a href='#crh-comment-Commit 5e52444eb7f67dc243247b5ad71161572ff015ef src/func/email.php 15 52'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/commit/5e52444eb7f67dc243247b5ad71161572ff015ef#commitcomment-9533083'>File: src/func/email.php:L52 (5e52444)</a></b>
- <a href='https://github.com/Phhere'></a> I think that the namespace needs to be fixed there too
- [x] <a href='#crh-comment-Commit 81b23018af8ed887de6b9dc84dafe36c035bcf3e myprofile.php 12 417'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/commit/81b23018af8ed887de6b9dc84dafe36c035bcf3e#commitcomment-9568700'>File: myprofile.php:L417 (81b2301)</a></b>
- <a href='https://github.com/Phhere'></a> echo is missing
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/205#issuecomment-73751681'>General Comment</a></b>
- <a href='https://github.com/Phhere'></a> I still think that php mailer shoul be inside the components folder to make clear that it is not our part of code
- <a href='https://github.com/derchrisuk'></a> I agree, but I got some include errors when the module was not in the same folder as the class
- <a href='https://github.com/Phhere'></a> But that should be resolveable ;) maybe a chdir into the components folder and than back after including
- <a href='https://github.com/derchrisuk'></a> :+1:
- <a href='https://github.com/Phhere'></a> :shipit:
- <a href='https://github.com/derchrisuk'></a> I think I saw a problem with mails not being delivered as HTML.
Need to check each email send, and what the actual email looks like.
Once done can be merged.
- [x] <a href='#crh-comment-Pull 24c20346947334ca2e354ab3fd071b48de6d6b12 composer.json 21'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/205#discussion_r25248170'>File: composer.json:L1-14</a></b>
- <a href='https://github.com/Pascalmh'></a> This should be the only `require` element, all codesniffer, test-reporter and phpunit are [`require-dev`](https://getcomposer.org/doc/04-schema.md#require-dev).

<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/205?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/205?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/205?approve=0'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL/pull/205'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by webSPELL-org** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL/pull/205'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by Phhere** [comment]
I still think that php mailer shoul be inside the components folder to make clear that it is not our part of code

**Comment by derchrisuk** [comment]
I agree, but I got some include errors when the module was not in the same folder as the class

**Comment by Phhere** [comment]
But that should be resolveable ;) maybe a chdir into the components folder and than back after including

**Comment by derchrisuk** [comment]
:+1:

File: composer.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+    "name": "webspell/webspell",
+    "description": "webSPELL CMS",
+    "keywords": ["cms", "gaming"],
+    "license": "GPL-3.0",
+    "type": "project",
+    "require": {
+        "squizlabs/php_codesniffer": "2.1.0",
+        "codeclimate/php-test-reporter": "dev-master",
+        "phpunit/phpunit": "4.4.0",
+        "phpmailer/phpmailer": "5.2.*"
```
Comment (Pascalmh) [inline composer.json]: This should be the only `require` element, all codesniffer, test-reporter and phpunit are [`require-dev`](https://getcomposer.org/doc/04-schema.md#require-dev).

**Comment by Phhere** [comment]
:shipit:

**Comment by derchrisuk** [comment]
I think I saw a problem with mails not being delivered as HTML.
Need to check each email send, and what the actual email looks like.
Once done can be merged.

**Comment by derchrisuk** [comment]
:+1:<a href='#crh-update-none'></a>

File: composer.json (hunk 1 above)
Comment (derchrisuk) [inline composer.json]: :+1:<a href='#crh-update-none'></a>

**Comment by derchrisuk** [comment]
:shipit:

**Pull request merged**